Source/WebCore/dom/Element.h

```diff
@@ -85,6 +85,7 @@ class Text;
 class UniqueElementData;
 class ValidatedFormListedElement;
 class WebAnimation;
+class WritingSuggestionData;
```

Comment:
Why do we need to add this here? I don’t see any use of this class name in this file. This must be left over from an earlier version of this patch.